modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/ViewConflictsMVCRenderCommand.java

```diff
@@ -101,6 +121,9 @@ public String render(
 		catch (SQLException sqlException) {
 			throw new ORMException(sqlException);
 		}
+		catch (IOException ioException) {
+			throw new PortletException(ioException);
+		}
```

Comment:
Usually we sort the catch clauses by exception name. However, since we're handling the PortalException and IOException the same, let's catch both exceptions in a single catch block.

```
catch (IOException | PortalException exception) {
```